### PR TITLE
Add wind velocity inputs to offline oil floats

### PR DESCRIPTION
**Pull request by dkobashi**
I've modified source code to add wind velocity inputs to offline oil floats.

### CODE REVIEW

File: ROMS/Modules/mod_clima.F [hunk 1]
```diff
@@ -304,6 +314,15 @@ SUBROUTINE allocate_clima (ng, LBi, UBi, LBj, UBj)
 #endif
 
 !Added by DJ ------------------------
+#if defined OFFLINE && FLOAT_OIL
```
Comment (kthyng) [inline ROMS/Modules/mod_clima.F:317]: #if defined OFFLINE && defined FLOAT_OIL

File: ROMS/Modules/mod_clima.F [hunk 2]
```diff
@@ -610,6 +629,21 @@ SUBROUTINE initialize_clima (ng, tile)
 !      END DO
 !#endif
 
+#if defined OFFLINE && defined_FLOAT_OIL
```
Comment (kthyng) [inline ROMS/Modules/mod_clima.F]: #if defined OFFLINE && defined FLOAT_OIL

**Review by kthyng — CHANGES_REQUESTED** [review]
Can you make the changes in the comments?

**Comment by kthyng** [comment]
Wait I think I looked at an older version of the code changes?

**Comment by dkobashi** [comment]
yes. After I did pull request, I found an error so I edited mod_clima.F and did pull request again. my apology.

**Comment by kthyng** [comment]
Ok, can you change mod_clima line 317? I think this one hasn't changed.

What is necessary to turn on to input wind then? And what if you don't want wind turned on?

**Comment by dkobashi** [comment]
If OFFLINE and FLOAT_OIL are both turned on, wind is included into Dmitry's
code.
If you want to name it as a def, just let me know. It sounds like Dmitry
wants to include it with FLOAT_OIL.

**Comment by kthyng** [comment]
> Ok, can you change mod_clima line 317? I think this one hasn't changed.

Can you make this change please?

> If OFFLINE and FLOAT_OIL are both turned on, wind is included into Dmitry's code. If you want to name it as a def, just let me know. It sounds like Dmitry wants to include it with FLOAT_OIL.

I'll let @DmitryDukh worry about this part I guess.

**Comment by kthyng** [comment]
I will merge and then fix the part with the typo.

**Pull request merged**